grammar.pegjs

```diff
@@ -35,7 +35,7 @@ selectors = s:selector ss:(_ "," _ selector)* {
 }
 
 selector
-  = a:sequence ops:(binaryOp sequence)* {
```

Comment:
This is too permissive. Instead, I would have just allowed an optional `binaryOp` in the `has` production.

Reply:
It can’t go directly in the has production, because that would cause problems if the second alternative has a root operator (`:has(> a, > b)`). I have resolved this in #145